mappings/net/minecraft/client/Minecraft.mapping

```diff
@@ -16,5 +26,8 @@ CLASS net/minecraft/client/Minecraft net/minecraft/client/Minecraft
 	METHOD a start (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)V
 		ARG 0 playerName
 	METHOD b getGameDirectory ()Ljava/io/File;
+	METHOD c printOpenGLError (Ljava/lang/String;)V
+	METHOD j toggleFullscreen ()V
+	METHOD l isLevelOnClient ()Z
```

Comment:
hasLevel?

Reply:
It also checks if the level is on the client which ugh yea I guess hasLevel is better

Reply:
```suggestion
	METHOD l hasLevel()Z
```